app/Http/Controllers/Api/ApiActivityController.php

```diff
@@ -140,7 +141,8 @@ public function update(Request $request, $activityId)
         JournalEntry::add($activity);
 
         // Get the attendees
-        $attendeesID = $request->get('contacts');
+        /** @var array */
```

Comment:
Why are you using this?

Reply:
Because it pleases psalm!

Reply:
(but I don't know why it does not detect it automatically)